_scss/pages/home.scss

```diff
@@ -5,6 +5,11 @@ body {
 .page--home {
   position: relative;
 
+  .footer {
+    position: relative;
+    z-index: -1000;
```

Comment:
do you need this if you're controlling it with js?

Reply:
This is so that it doesn't overlap with the nav... which I'm now realizing it still does on the project pages because those are absolutely positioned because they need to stay within the 100vh of the prev/next section...A PROBLEM FOR ANOTHER BRANCH AND ANOTHER PR